initial_pose: 
  x: 0.3
  y: 0.3
  z: 1
  R_deg: 0
  P_deg: 0
  Y_deg: 0
  camera_deg: -82
collision_distance: 0.85